### PR TITLE
AG-300 updated json keys

### PR DESCRIPTION
**Pull request by sagely1**

### CODE REVIEW

File: src/app/testing/gene-mocks.ts [hunk 1]
```diff
@@ -90,101 +86,100 @@ export const geneMock1: Gene = {
       initial_nomination: 2018,
     },
   ],
-  medianexpression: [
+  median_expression: [
     {
       ensembl_gene_id: 'ENSG00000147065',
```
Comment (hallieswan) [inline src/app/testing/gene-mocks.ts]: The ticket notes that `medianExpression.ensembl_gene_id` was removed from `median_expression` -- should these mocks be updated to remove`ensembl_gene_id` from `median_expression`?

File: src/app/models/genes.ts [hunk 2]
```diff
@@ -29,17 +28,16 @@ export interface NominatedTarget {
 
 export interface MedianExpression {
   ensembl_gene_id: string;
```
Comment (hallieswan) [inline src/app/models/genes.ts]: Should `ensemble_gene_id` also be removed from this interface?

Comment (sagely1) [reply src/app/models/genes.ts]: Good catch!  I only removed it on the api side.

File: src/app/testing/gene-mocks.ts (hunk 1 above)
Comment (sagely1) [reply src/app/testing/gene-mocks.ts]: yes, intellisense will demand it. 👍

**Review by hallieswan — APPROVED** [review]
Looks good to me!

**Pull request merged**